### PR TITLE
Added bank-of-anthos application support for pre-defined workflow

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>

**What this PR does / why we need it**:

 - Support for bank-of-anthos-chaos pre-defined workflow application
 - Added Bank-Of-Anthos Application Kubernetes manifests for weak and resilient cases.

**Special notes for your reviewer**:

### CODE REVIEW

File: custom/workflow-helper/app-deployer/app-manifest/resilient-bank-of-anthos.yaml [hunk 1]
```diff
@@ -0,0 +1,736 @@
...
...Q1QzNW1ybHQxZzljTGo0RllMb1drRAo4dU1hQjY1RFc4US9IN0gzR3ViRGxSSVovN2JVaHJVaVVuU1dsSUppdHVKclZxRVdYdDF6bHhtR2pHekt0dHRlCjNwYy9IOS96Y3FZdC9mdHdzdWJJWWtadkVCSUFBbmhaTnZCSjYva1liczFESVdONXZlRE1DOWU3eWY2ZmQzOVEKYWtPQWQxa0NnZ0VCQUpCQ2RraFVydHBJSWg4eWgwanpRa2M5QWZESlZBZ3k2MTE1cUJDS2YveTJzK1dONEhOdQpFdTROQ2hmVHFvc1phRXVDdDdVQlRla3ZnaUVDcVltN2NMRnlMQWVobmJTeXpnVHVDRGF3MDc5cXBhZGlHREJjCjI3VXFQaDgzWFJwTVJrSVJSUzd1TkxWY0FYZTNBYmdtSWdlWExvQnRmNVo2SkZxSURLRE5WMFk5VWJVRi9MTE8KQTBoRGU3SnhHSUdWWmVVaU1ZU3RqdFQzMytkZVF6clVtL0p0a09XSUpnZFN6YnYrQnZWc3dua2hkVmtjcDlJWApxSm9jQ3lua2VDN1pUUXdCa1psZ3NmMEx4SW1kNzVlemhKb1dqd2FraksrWnpwYk1Gb01KUmduVmYyOFIrdERhCktCeGQwVmowQUM1ZXBLTVQ2ajVGRy8vRDhkTDUwVjZmOGU4Q2dnRUJBTWgreklJb3FKZWlXTGgzOUFtTTRQZEkKSFNod0lISXBUNUhGS05jOTFzRnRDZFdXVDIweXVPZHNLZU9TTDZjMjJsVFpjcklFZFlDY0Q5SVhyWm8xVmxXOAorSGFmNDFhcHhlaU1PZVBNR3ZlNEQ2ZEdrUXcyWXBuNXlOQUUwYk1qa1hFU3JuMzMvV3JqSFFtWFdvNmFSdnVZCnkrMktzbGwvK1BzajlwNXd3WVYxRGp0QzFXU0lmdFZNMkJuRlViQVVyMmNHSVNXWlNnWU9yUDU1elZKK3REeloKSXhrMDZ6TUNZU3pEL1pPTUJZSjUvTFFPWnlPUUs0R2xnMytUUUNXUERHNTRtUEo0dElmNUZCZ25wSU5QRGNLQwpLc2RTQlg4S0xNeU83aEVzOCtuZFRWNWlxa0pscGNCdVJ3VXVUSmlFakc0MWZnaXZPVThTR0V5elZPbW1NNG89Ci0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg==
```
Comment (ispeakc0de) [inline custom/workflow-helper/app-deployer/app-manifest/resilient-bank-of-anthos.yaml:8]: Do we want to add secret data publicly?

File: custom/workflow-helper/app-deployer/app-manifest/resilient-bank-of-anthos.yaml [hunk 2]
```diff
@@ -0,0 +1,736 @@
...
+apiVersion: apps/v1
+metadata:
+  name: accounts-db
+  labels:
+    app: bank-of-anthos
+    name: accounts-db
+    tier: db
+spec:
+  serviceName: accounts-db
+  replicas: 1
```
Comment (ispeakc0de) [inline custom/workflow-helper/app-deployer/app-manifest/resilient-bank-of-anthos.yaml:49]: what is replica count for the resilient case?

File: custom/workflow-helper/app-deployer/app-manifest/resilient-bank-of-anthos.yaml (hunk 1 above)
Comment (oumkale) [reply custom/workflow-helper/app-deployer/app-manifest/resilient-bank-of-anthos.yaml:8]: Yes this is default one @ispeakc0de

File: custom/workflow-helper/app-deployer/app-manifest/resilient-bank-of-anthos.yaml (hunk 2 above)
Comment (oumkale) [reply custom/workflow-helper/app-deployer/app-manifest/resilient-bank-of-anthos.yaml:49]: We want 2 only for chaos pod.

**Review by ispeakc0de — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**